pygac/tests/test_reader.py

```diff
@@ -192,9 +192,50 @@ def test_get_tle_lines(self, read_tle_file, *mocks):
                 self.assertEqual(tle1, tle_data[tle_idx])
                 self.assertEqual(tle2, tle_data[tle_idx + 1])
 
+    @mock.patch('pygac.gac_reader.GACReader._get_corrupt_mask')
+    def test_get_angles(self, get_corrupt_mask):
+        """Test get_angles function of the reader."""
+        # Line: 1, 649, 6198 and 12658 from Tiros-N file (1980-01-03 11:47)
+        lon_i = np.array(
+            [69.41555, 152.10587, 164.3131, 67.23855])[:, np.newaxis]
+        lat_i = np.array(
+            [71.6283, 85.24265, -62.076958, 82.72296])[:, np.newaxis]
+        get_corrupt_mask.return_value = np.isnan(lon_i)
```

Comment:
It would be nice if you added a `np.nan` to `lon_i`. Then the masking can be tested as well.

Reply:
I tried that first but failed. If I add nan:s now I just get nan as output also for the angles. Should I do that?

Reply:
Sorry, that was not correct. What I meant was, if `get_corrupt_mask.return_value = np.array([0, 0, 0, 1]).astype(bool)` then you can also check that the last element in the angles arrays is `np.nan`

Reply:
Ok!